helper-cli/src/main/kotlin/commands/packagecuration/CreatePackageCurationCommand.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands.packagecuration
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.UsageError
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
+
+import java.io.IOException
+
+import org.ossreviewtoolkit.helper.common.getSplitCurationFile
+import org.ossreviewtoolkit.model.FileFormat
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.PackageCuration
+import org.ossreviewtoolkit.model.PackageCurationData
+import org.ossreviewtoolkit.model.writeValue
+import org.ossreviewtoolkit.utils.expandTilde
+
+internal class CreatePackageCurationCommand : CliktCommand(
+    help = "Create a single curation file for a package id complying to the directory structure created by the" +
+            "`split-curations` sub-command."
+) {
+    private val packageId by option(
+        "--package-id",
+        help = "The target package id for which a curation should be generated."
+    ).convert { Identifier(it) }
+        .required()
+
+    private val outputDir by option(
+        "--package-curations-dir",
+        help = "The output package curations directory."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = false, canBeDir = true, mustBeWritable = true, mustBeReadable = false)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    override fun run() {
+        val outputFile = getSplitCurationFile(outputDir, packageId, FileFormat.YAML.fileExtension)
+
+        if (outputFile.exists()) {
+            throw UsageError("File ${outputFile.absolutePath} already exists.")
+        }
+
+        try {
+            outputFile.parentFile.mkdirs()
+            outputFile.writeValue(listOf(
+                // Adding an empty comment to avoid `{}` for an empty PackageCurationData.
+                PackageCuration(packageId, PackageCurationData(comment = ""))
+            ))
+        } catch (e: IOException) {
+            throw IOException("Failed to create ${outputFile.absoluteFile}.", e)
```

Comment:
Not sure if it makes sense. Have you checked if throwing ProgramResult fits ?

Reply:
Clikt's `ProgramResult` is not an Exception. It only allows me to set the `statusCode` the exception would therefore be lost.
As throwing an `IOException` already results in the statusCode `1`, I think this is the better way to handle this.